frontend/src/metabase/visualizations/echarts/pie/option.ts

```diff
@@ -0,0 +1,42 @@
+import type { EChartsOption, RegisteredSeriesOption } from "echarts";
+
+import type {
+  ComputedVisualizationSettings,
+  RenderingContext,
+} from "metabase/visualizations/types";
+
+import type { PieChartModel } from "./model/types";
+
+// static, non-computed values
+const SUNBURST_SERIES_OPTIONS: RegisteredSeriesOption["sunburst"] = {
+  type: "sunburst",
+  sort: undefined,
+  label: {
+    rotate: 0,
+    overflow: "none",
+    lineHeight: 50, // TODO update this later
```

Comment:
Will get the exact values later when polishing, and will confirm it looks good with product+design